.circleci/config.yml

```diff
@@ -577,21 +628,69 @@ jobs:
           name: Generate SQL content
           command: |
             export PATH="venv/bin:$PATH"
-            ./script/bqetl generate all \
-              --target-project moz-fx-data-shared-prod
+
+            # Check if the generated-sql branch can simply be re-used and SQL generation can be skipped.
```

Comment:
Similar comments as above, and a few comment typos below. Maybe some of this fits into a `.sh` script similar to https://github.com/mozilla/lookml-generator/tree/main/bin for re-use or even better a python script 😄

Reply:
I think for now it's probably best to be explicit with what is happening in the CI. If everything ends up working fine then maybe we can consider a script